docs/source/technical_features.md

```diff
@@ -37,10 +60,11 @@ the prior knowledge about the subjective lesion borders is then leveraged to the
 ![Figure Data Augmentation on lesion ground truths](https://github.com/neuropoly/ivado-medical-imaging/raw/master/images/dilate-gt.png)
 
 ## Network architectures
-- [U-net](https://arxiv.org/pdf/1505.04597.pdf)
-- [HeMIS](https://arxiv.org/abs/1607.05194)
+- [UNet](https://arxiv.org/pdf/1505.04597.pdf), with control of the network depth.
+- HeMIS-UNet: integrates the [HeMIS](https://arxiv.org/abs/1607.05194) strategy to deal with missing modalities within a UNet training scheme.
+- FiLMed-UNet, based on [FiLM](https://arxiv.org/pdf/1709.07871.pdf) strategy adapted to the [segmentation task](#physic-informed-network).
 
 ## Loss functions
-- [Dice Loss](https://arxiv.org/abs/1606.04797)
-- [Focal Loss](https://arxiv.org/pdf/1708.02002.pdf)
-- [Generalised Dice Loss](https://arxiv.org/pdf/1707.03237.pdf)
+- [Dice Loss](https://arxiv.org/abs/1606.04797). Also adapted for multi-label segmentation tasks, by averaging the loss for each class.
+- [Focal Loss](https://arxiv.org/pdf/1708.02002.pdf). Additional possibility to be combined with the DiceLoss is a linear combination.
+- [Generalised Dice Loss](https://arxiv.org/pdf/1707.03237.pdf). In particular, for cases of high class imbalance, the background volume is here accounted by a weighting of the inverse of its volume.
```

Comment:
I've rephrased-- please check it's ok. 
```suggestion
- [Generalised Dice Loss](https://arxiv.org/abs/1707.03237): The background volume is weighted by the inverse of its area. This loss is particularly interesting with class imbalance.
```

Reply:
Thanks @jcohenadad.
What I am trying to do here is to highlight an addition compared to the original paper, [new commit here](https://github.com/neuropoly/ivado-medical-imaging/pull/277/commits/c7bae270e53156f218fa7ef0bbcf367c551fa49b).